### ISSUE TITLE
Option to turn opaque background into transparent background

### ISSUE THREAD

**Issue by phbernard** [RealFaviconGenerator/realfavicongenerator#45]
It can be useful to turn an opaque picture to a picture with a transparent background. This would be useful for:
- Desktop favicon, for better integration in tabs
- Windows 8 tiles.

**Comment by MoonSweep** [comment]
I agree. I have a very simple white on black favicon and it would be nice to automatically turn the black regions to transparent for Windows 8 tiles.
